### PR TITLE
New strapi component that allows continue button and radio button combo

### PR DESCRIPTION
**Pull request by TasTasniem**

### CODE REVIEW

File: Beis.LearningPlatform.Web/Models/CmsNextPageRadioButtonViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        public CmsNextPageRadioButtonViewModel(CMSPageComponent cmsPageComponent)
+        {
+            _cmsPageComponent = cmsPageComponent;
+        }
+
+        public bool HasContent
+        {
+            get
+            {
+                return !string.IsNullOrEmpty(_cmsPageComponent.Radio1Text)
```
Comment (emailtokvk) [inline Beis.LearningPlatform.Web/Models/CmsNextPageRadioButtonViewModel.cs:21]: Is Whitespace allowed?

**Review by emailtokvk — CHANGES_REQUESTED** [review]

File: Beis.LearningPlatform.Web/Models/CmsNextPageRadioButtonViewModel.cs (hunk 1 above)
Comment (TasTasniem) [reply Beis.LearningPlatform.Web/Models/CmsNextPageRadioButtonViewModel.cs:21]: Don't know. Let me know asap. Need to get this out in this sprint

Comment (emailtokvk) [reply Beis.LearningPlatform.Web/Models/CmsNextPageRadioButtonViewModel.cs:21]: I can't say. If the whitespace is not allowed then please change it to string.IsNullOrWhiteSpace where ever there is NullOrEmpty

File: Beis.LearningPlatform.Web/wwwroot/js/site.js [hunk 2]
```diff
@@ -94,4 +94,30 @@ for (i = 0; i < acc.length; i++) {
...
     });
-}
\ No newline at end of file
+}
+
+//cms_component_NextPageRadioButton
+$(document).ready(function () {
+    $("#cms_component_NextPageRadioButton_divjsenabled").attr("style", "");
+});
+$('input[type=radio][name=cms_component_NextPageRadioButton_rdon]').change(function () {
```
Comment (JamesChristianson) [inline Beis.LearningPlatform.Web/wwwroot/js/site.js:103]: We dont use jQuery - please remove

File: Beis.LearningPlatform.Web/Views/Shared/Components/CmsNextPageRadioButton/Default.cshtml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+
+                    <p class="govuk-button-group">
+                    </p>
+
+
+                    <div class="  ">
+                        <div>
+                            <span class="beis-link-icon">
+                                <img src="/assets/images/link-arrow.svg" class="svgimg beis-link-icon " width="23" height="24">
```
Comment (JamesChristianson) [inline Beis.LearningPlatform.Web/Views/Shared/Components/CmsNextPageRadioButton/Default.cshtml:69]: please make repested items into components e.g. this link arrow is repeated frequently

**Review by JamesChristianson — CHANGES_REQUESTED** [review]

**Review by jdssunil — DISMISSED** [review]

File: Beis.LearningPlatform.Web/wwwroot/js/site.js (hunk 2 above)
Comment (TasTasniem) [reply Beis.LearningPlatform.Web/wwwroot/js/site.js]: DONE in last commit
Changed to pure javascript

File: Beis.LearningPlatform.Web/Views/Shared/Components/CmsNextPageRadioButton/Default.cshtml (hunk 3 above)
Comment (TasTasniem) [reply Beis.LearningPlatform.Web/Views/Shared/Components/CmsNextPageRadioButton/Default.cshtml]: DONE in last commit
Created new component

**Review by emailtokvk — APPROVED** [review]

**Review by JamesChristianson — DISMISSED** [review]

**Review by rajchala — APPROVED** [review]
Tested on DEV

**Pull request merged**